valkey.conf

```diff
@@ -513,6 +513,9 @@ rdb-del-sync-files no
 #
 # The Append Only File will also be created inside this directory.
 #
+# The Cluster config file be written inside this directory, if 'cluster-config-file'
+# configuration directive is not an absolute path.
```

Comment:
If it's a relative path, it is relative to the `dir`, even if it points outside the dir, right? For example `cluster-config-file ../foo/nodes.conf`.

```suggestion
# The Cluster config file is written relative this directory, if the
# 'cluster-config-file' configuration directive is a relative path.
```

Reply:
yes, that is right